app/config.py

```
@@ -0,0 +1,89 @@
+            )
+
+        headers = [header.strip() for header in self.allowed_headers.split(",")]
+        headers = [header for header in headers if header]
+
+        if not headers:
+            raise ValueError("ALLOWED_HEADERS must contain at least one header name")
+
+        return headers
+
+
+@lru_cache()
```

Ruff (0.14.10)

Unnecessary parentheses to `functools.lru_cache`

Remove unnecessary parentheses

(UP011)